textproc/tesseract/Portfile

```diff
@@ -25,26 +39,36 @@ maintainers         {mark @markemer} openmaintainer
 
 description         Open source OCR engine
 
-long_description    The Tesseract OCR engine was one of the top  3  engines in \
-                    the 1995 UNLV Accuracy test.  Between 1995 and 2006 it had \
-                    little work done on it,  but it is  probably  one  of  the \
-                    most accurate  open  source  OCR  engines  available.  The \
-                    source code will read a binary, grey or  color  image  and \
-                    output text.  A tiff reader is built  in  that  will  read \
-                    uncompressed TIFF images,  or libtiff can be added to read \
-                    compressed images.
+long_description    This package contains an OCR engine - libtesseract and a command line program - tesseract. \
+                    Tesseract 4 adds a new neural net (LSTM) based OCR engine which is focused on line recognition, \
+                    but also still supports the legacy Tesseract OCR engine of Tesseract 3 which works by recognizing \
+                    character patterns.
 
 if {${subport} eq ${name}} {
-    revision                2
-
     github.livecheck.regex  {(\d\.\d+(\.\d+)?(?!-rc))}
 } else {
     livecheck.type          none
 }
 
-subport ${name}-training {
-    revision                2
+if {${subport} in [list tesseract tesseract-training]} {
+    notes "To use tesseract you must also install one of its language data subports. (ex tesseract-eng)"
```

Comment:
You might want to break this up into two lines. Many people will be viewing messages in an 80 column window.

Reply:
Will do, I'll add an 80 column line to my editor too.